test/runtests.jl

```diff
@@ -111,6 +111,7 @@ end
 if "CUDA" in ARGS
     @safetestset "GPU - cuda" begin @time include("gpu/cuda.jl") end
     @safetestset "GPU - data" begin @time include("DataLayouts/cuda.jl") end
+    @safetestset "GPU - grids" begin @time include("Grids/grids.jl") end
```

Comment:
We don't run `runtests.jl` on buildkite (https://github.com/CliMA/ClimaCore.jl/issues/1290), so we need to add a buildkite job to exercise this.